docs/source/api/components/workspace.rst

```diff
@@ -0,0 +1,35 @@
+Workspace
+=========
+
+Base class
+----------
+
+.. autoclass:: tango.workspace.Workspace
+   :members:
+
+Implementations
+---------------
+
+.. autoclass:: tango.workspace.MemoryWorkspace
+   :members:
+
+.. autoclass:: tango.local_workspace.LocalWorkspace
+   :members:
```

Comment:
Consider not documenting the members since those are all documented in the base class, and if you don't write new docstrings, the docstrings will be inherited from the base class, which can lead to confusing / misleading docs.
```suggestion
.. autoclass:: tango.workspace.MemoryWorkspace

.. autoclass:: tango.local_workspace.LocalWorkspace
```

Reply:
That's better, but why do the `StepCache` classes not need that?